develop/dev-guide-sample-application-python-sqlalchemy.md

```diff
@@ -76,6 +76,10 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
 <SimpleTab>
 <div label="TiDB Serverless">
 
+> **Note:**
+>
+> Currently TiDB Serverless clusters come with a limitation that if there is no active connections for 5 minutes, they will shutdown themselves and all connections will be closed. So when using SQLAlchemy against TiDB Serverless clusters, the pooled connections might get `OperationalError` like `Lost connection to MySQL server during query` or `MySQL Connection not available`. You can set `pool_recycle` parameter to `300` to avoid this. For more information about it, refer to [Dealing with Disconnects](https://docs.sqlalchemy.org/en/20/core/pooling.html#dealing-with-disconnects).
```

Comment:
```suggestion
> Currently, TiDB Serverless clusters have a limitation: if there are no active connections for 5 minutes, they will shut down, which closes all connections. Therefore, when using SQLAlchemy with TiDB Serverless clusters, pooled connections might encounter `OperationalError` such as `Lost connection to MySQL server during query` or `MySQL Connection not available`. To avoid this error, you can set the `pool_recycle` parameter to `300`. For more information, see [Dealing with Disconnects](https://docs.sqlalchemy.org/en/20/core/pooling.html#dealing-with-disconnects) in SQLAlchemy documentation.

```